github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -284,7 +525,7 @@ presubmits:
           resources:
             requests:
               memory: "34Gi"
-  
+
```

Comment:
Unrelated change

Reply:
So you are saying I should move the removal of trailing whitespaces to a new PR?

Reply:
No :)
I am not clear what triggered this... is it your editor?

Reply:
Yes, IntelliJ has a switch that automatically removes trailing whitespace.